README.md

```diff
@@ -1,2 +1,37 @@
 # credentials-io
 Library for reading/writing credentials
+
+## API
+
+### verify(identity, options, callback)
```

Comment:
What does an identity object look like? What are the options you can pass the function? What's the function signature for the callback - callback(err, identity)?

Look at this for an example on how to document parameters: https://github.com/digitalbazaar/credentials-polyfill/blob/master/README.md#registering-a-decentralized-identifier